app/components/admin/stats/budget_supporting_component.rb

```diff
@@ -0,0 +1,31 @@
+class Admin::Stats::BudgetSupportingComponent < ApplicationComponent
+  attr_reader :budget
+
+  def initialize(budget)
+    @budget = budget
+  end
+
+  private
+
+    def stats
+      @stats ||= Budget::Stats.new(budget)
+    end
+
+    def headings_stats
+      @headings_stats ||= stats.headings
+    end
+
+    def vote_count
```

Comment:
Trying not to think that the supports are in the `Vote` table and the votes in the `Ballot::Line` table 😀 , what would you think if this method `vote_count` inside the Budget**Supporting**Component that calls a stats.total_**supports** we call it "**support**_count" 🤔

Reply:
Yes, there's quite a mess with supports, votes and ballots in investments :sweat_smile:.

The main argument I see against this change is that `vote_count` is consistent with the `admin.stats.budget_supporting.vote_count` translation key we're using, and unfortunately renaming the translation key might result in losing the translations :relieved:.

No strong preference, but I vote for keeping `vote_count` for now.

What do you think?